itn-eval-metrics.Rmd

```diff
@@ -535,10 +536,34 @@ ggplot2::ggsave(file.path("plots", "itn_software_website_traffic.png"), width =
 ```{r}
  #download_stats <- cran_stats(c("ottrpal", "conrad", "ari", "text2speech"))
 
- #download_stats %>% dplyr::summarize(download_total = sum(downloads))
+download_stats <- cranlogs::cran_downloads(packages = c("ottrpal", "conrad", "ari", "text2speech"), from = "2017-08-31", to = "last-day") #first version of ari published on 2017-08-31
 
- #download_stats %>% dplyr::group_by(package) %>%
- #  dplyr::summarize(download_total = sum(downloads))
+download_stats %>% dplyr::summarize(download_total = sum(count))
+
+download_stats %>% dplyr::group_by(package) %>%
+   dplyr::summarize(download_total = sum(count))
+
+download_stats_to_plot <- download_stats %>% separate(date, into=c("year", "month name", "day"), sep = "-") %>% unite("Month", c("year", "month name"), sep='-', remove=TRUE) %>%  #cran_downloads returns the daily downloads, and so doing some prep work to turn that into monthly downloads
+  group_by(Month, package) %>% summarise(monthly_downloads = sum(count)) %>% #summarize monthly downloads by package
+  filter(monthly_downloads > 0) #drop the 0's 
+
+xlabel_view <- c(rep(c("black", "transparent", "transparent", "transparent"), 41), "black", "transparent") #166 rows
+cc <- rev(c("#fde725", "#addc30", "#5ec962", "#28ae80", "#21918c", "#2c728e", "#3b528b", "#472d7b", "#440154"))
+viridis_cc <- c("#440154", "#2c728e", "#28ae80", "#addc30")
+
+ggplot(download_stats_to_plot, aes(Month, monthly_downloads, group=package, color = package)) + 
+  geom_line() + 
+  geom_point() +
+  scale_colour_manual(values=viridis_cc) +
+  theme(panel.background = element_blank(), panel.grid = element_blank()) +
+  geom_vline(aes(xintercept = "2019-05"), linetype='dashed', color = '#addc30') + #text2speech published date
```

Comment:
Love this!

Reply:
We should add these same lines for when courses were published. (again that can be on another branch)

Reply:
I can work on that!